src/js/components/Button/StyledButtonKind.js

```diff
@@ -70,21 +70,18 @@ const basicStyle = props => css`
 `;
 
 // CSS for this sub-object in the theme
-const kindPartStyles = (obj, theme, colorValue) => {
+export const kindPartStyles = (obj, theme, colorValue) => {
```

Comment:
should we move this to utils?

Reply:
Yes, I can move it to utils -- possible that in future we would leverage it for other component styles.

Reply:
moved to utils in recent commit